<html>

<head>
<style type="text/css">
.knitr .inline {
  background-color: #f7f7f7;
  border:solid 1px #B0B0B0;
}
.error {
	font-weight: bold;
	color: #FF0000;
}
.warning {
	font-weight: bold;
}
.message {
	font-style: italic;
}
.source, .output, .warning, .error, .message {
	padding: 0 1em;
  border:solid 1px #F7F7F7;
}
.source {
  background-color: #f5f5f5;
}
.rimage .left {
  text-align: left;
}
.rimage .right {
  text-align: right;
}
.rimage .center {
  text-align: center;
}
.hl.num {
  color: #AF0F91;
}
.hl.str {
  color: #317ECC;
}
.hl.com {
  color: #AD95AF;
  font-style: italic;
}
.hl.opt {
  color: #000000;
}
.hl.std {
  color: #585858;
}
.hl.kwa {
  color: #295F94;
  font-weight: bold;
}
.hl.kwb {
  color: #B05A65;
}
.hl.kwc {
  color: #55aa55;
}
.hl.kwd {
  color: #BC5A65;
  font-weight: bold;
}
</style>


<script type="text/javascript">
  
  (function() {
    // http://stackoverflow.com/questions/10964966/detect-ie-version-in-javascript
    function isIE () {
      var myNav = navigator.userAgent.toLowerCase();
      return (myNav.indexOf('msie') != -1) ? parseInt(myNav.split('msie')[1]) : false;
    }
    
    if (isIE() === 8 || isIE() === 9){
      // Set an absolute base href.
      baseURL = document.URL;
      baseURL = baseURL.substring(0, baseURL.lastIndexOf('/'));
      document.getElementsByTagName('base')[0].setAttribute('href', baseURL + '/_w_83ad5705/');
    }
  })();
</script>


<title>Relative frequency measurements: Metrics for sample quality, sequencing integrity, and batch effects in targeted NGS.</title>
<link rel="stylesheet" type = "text/css" href="bootstrap.css">
</head>
<style>
  .black_overlay{
    display: none;
    position: absolute;
    top: 0%;
    left: 0%;
    width: 100%;
    height: 100%;
    background-color: black;
    z-index:1001;
    -moz-opacity: 0.8;
    opacity:.80;
    filter: alpha(opacity=80);
}
.white_content {
    display: none;
    position: absolute;
    top: 10%;
    left: 10%;
    width: 80%;
    height: 80%;
    padding: 16px;
    border: 16px solid orange;
    background-color: white;
    z-index:1002;
    overflow: auto;
}
</style>

<body>

<h1 align="center" style="background-color:#ffd0a1"> Relative frequency measurements: Metrics for sample quality, sequencing integrity, and batch effects in targeted NGS.</h1>
<div class="row">
  <div class="col-sm-4">
    <img src="./ASA_Poster_Content/images/HTG-Logo-top.png" style="height:45px;display:block;margin:auto">
  </div>
  
  <div class="col-sm-4">
    <h5 align="center"> Bonnie LaFleur<sup>1</sup>, Dominic LaRoche<sup>1,2</sup>, Kurt Michels<sup>1,2</sup>, Shripad Sinari<sup>2</sup>,  Dean Billhiemer<sup>2</sup></h5>
    <h6 align="center"> <sup>1</sup>HTG Molecular Diagnostics, Inc.,  <sup>2</sup>University of Arizona, Tucson AZ</h6>
  </div>
  
  <div class="col-sm-4">
    <img src="./ASA_Poster_Content/images/UA_CPH_RGB_Primary.png" style="height:45px;display:block;margin:auto">
  </div>
</div>
<!--<hr width="100%" size="10" color="orange" noshade>-->
<!---Main Contents-->
<div class="row">
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-2"><div class="rcode"><!--html_preserve-->

<h4>Next-Generation Sequencing Data is Compositional</h4>
    
    <ul style="font-size:12px">
      <li style="margin-bottom:20px">Next-generation sequencing measurements provide only <i>relative</i> abundances of transcripts</li>
      <li >Reasons NGS data are compositions (Lovell et al. 2011).</li>
          <ul>
            <li>Finite sample volume from which DNA/RNA is extracted</li>
            <li>Finite number of reads available on each platform (as the number of reads increases for 1 transcript the number of available reads for the other transcripts decreases, Fig. 1)</li>
            <li>Read depth (number of reads allocated to a sample) varies from run-to-run and sample-to-sample</li>
          </ul>

      <li style="margin-top:20px">Traditional methods can produce misleading results for compositional data</li>
        <ul>
          <li>Spurious correlation (Pearson 1896)</li>
          <li>Univariate results may be misleading (Lovell et al. 2015)</li>
          <li>Negative bias of the covariance matrix (Aitchison 1985)</li>
        </ul>
      
    </ul>
<p align="right"> <a href = "javascript:void(0)" onclick = "document.getElementById('content1').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>



<!--/html_preserve--></div></div>
    </form>
  </div>
  
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-3"><div class="rcode"><!--html_preserve--><h4>Properties of Compositional Data</h4>
  
<ul style="font-size:12px">
  <li>Relative abundance data arise from a sum constraint: \(\sum_{i=1}^{D} t_i = \mathbb{T}\)</li>
  <ul>
    <li>\(t_i\) gives the number of reads to sample \(i\) in a sequencing run with \(D\) samples.</li>
    <li>The sum constraint, \(\mathbb{T}\), can change between platforms, sequencing runs, and samples.</li>
    <li>This leads to an arbitrary difference in <i>scale</i> between samples.</li>
  </ul>
  <li style="margin-top:15px">Compositions lie in the \(D-1\) dimensional simplex (\(S^{D-1}\)).  As a result of the constraints on compositional data, the covariance matrix of an un-transformed composition is difficult to interpret.</li>
</ul>
 
<div class="row">
  <div class="col-sm-6">    
  <img src="./ASA_Poster_Content/images/BarAnimation.gif" alt="HTML5 Icon" style="width:200px;height:120px;"> 
  </div>
  <div class="col-sm-6">
   <p style="font-size:12px">
   <b>Figure 1:</b> The impact of changes to a single transcript on the relative abundance of the other transcripts being measured. Ignoring the constraints imposed by compositional data can lead to spurious results.
   </p>
  </div>
</div>

<!--/html_preserve--></div></div>
    </form>
  </div>
  
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-4"><div class="rcode"><!--html_preserve--><h4>Compositional Operations and their Properties</h4>
<p style="font-size:10px">Many problems induced by compositional data can be alleviated by working with <i><b>log-ratios</b></i> of components. John Aitchison (1985) outlined several operations for working with compositions:</p>
<ul style="font-size:10px">
  <li>Additive Log-Ratio (ALR): \( ALR (x_i) = log (\frac{x_i}{x_D})  \), where \(x_D\) is the \(D^{th}\) component of the vector X.</li>
    <ul>
      <li>Interpretable variance-covariance matrix</li>
      <li>Does not treat all components equally as one component is used for denominator</li>
    </ul>
  <li>Centered Log-Ratio (CLR): \( CLR (x_i) = log( \frac{x_i}{g(X)}) \), where \(g(X)\) is the geometric mean of X.</li>
    <ul>
      <li>Interpretable variance-covariance matrix</li>
      <li>Preserves the dimension of the original composition</li>
      <li>Resulting variance-covariance matrix is singular</li>
    </ul>
</ul>
<div class="row">
  <div class="col-sm-6">
    <img src="ASA_Poster_Content/images/ALRAnimation.gif" style="width:200px;height:120px;">
  </div>
  <div class="col-sm-6">
    <p style="font-size:10px"><b>Figure 3.</b>The ALR transformation reduces the dimension of the composition but preserves the absolute relationships between the components.  Transformed data can then be used with standard multivariate methods.</p>
<!--    <p align="right">   <a href = "javascript:void(0)" onclick = "document.getElementById('content3').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>
-->
  </div>
</div><!--/html_preserve--></div></div>
    
    </form>
  </div>
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-5"><div class="rcode"><!--html_preserve--><h4>Counts per Million: A compositional operation</h4>
<ul style="font-size:12px">
  <li>Counts per million (CPM) transformation is a widely used standardization method
    \[log_2 (CPM(x_i)) = log_2(\frac{x_i+0.5}{(\sum_{j=1}^K x_j)+1} \times 10^6) \]
    where, \(x_i\) is the read count for gene <i>i</i> in a sample with <i>K</i> genes measured.
  </li> 
</ul>
<ul style="font-size:12px">
  <li>What does CPM accomplish?</li>
    <ul>
      <li>CPM simply re-scales the data, a compositional <i><b>closure</b></i> operation.</li>
      <li>Reads from each sample then sum to \(\approx\)1 million</li>
    </ul>
  <li>What does CPM <i>not</i> accomplish?</li>
    <ul>
      <li>Data still resides in the compositional simplex</li>
      <li>Traditional methods may still produce spurious results</li>
        <ul>
          <li>Spurious correlation</li>
          <li>Negatively biased covariance matrix</li>
          <li>etc.</li>
        </ul>
    </ul><!--/html_preserve--></div></div>
    
    </form>
  </div>
</div>

<div class="row">
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-6"><div class="rcode"><!--html_preserve--><h4>Composition-Based Quality Control</h4>
<p style="font-size:10px">We capitalize on the compositional nature of RNA-seq data to develop a quality control metric for sequencing runs</p>
<ul style="font-size:10px">
  <li>The number of reads/sample is constrained by total reads/sequencing run</li>
  <li>We expect the proportion of reads each sample is allocated to be apprx. equal</li>
  <li>CLR transformation allows the use of standard outlier detection methods (Cook's Distance/ GOF)
</ul>
<img src="./ASA_Poster_Content/images/CLR_QC.png" style="width:400px;height:195;">
<div class="col-sm-10">
  <p style="font-size:10px"><b>Figure 4.</b>Heatmap of CLR transformed reads showing the samples with an outlying proportion of the total reads in a sequencing run.</p>
</div>
<div class="col-sm-2">
  <p align="right">
    <a href = "javascript:void(0)" onclick = "document.getElementById('content5').style.display='block';document.getElementById('fade').style.display='block'">Expand</a>
  </p>
</div>

<!--/html_preserve--></div></div>
    </form>
  </div>
  
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-7"><div class="rcode"><!--html_preserve--><h4>Sub-compositions and Amalgamations</h4>
<p style="font-size:10px">Recognizing the compositional nature of sequencing data leads to several useful properties:</p>
<ul style="font-size:10px">
  <li>Sub-Compositions</li>
    <ul>
      <li>Ability to isolate sub-groups of probes from the sequenced gene set for focused analysis</li>
      <li>Can reduce noise from excluded genes with high variation</li>
    </ul>
  <li>Amalgamations</li>
    <ul>
      <li>Can collapse groups of genes with the same functionality into a single proportion</li>
      <li>Facilitates the use of exploratory techniques for finding differences in pathways/functional groups</li>
      <li>Amalgamations retain compositional structure</li>
    </ul>  
</ul>
<img src="ASA_Poster_Content/images/hierarchicalCluster.png" style="height:120px">
<p align="right"> <a href = "javascript:void(0)" onclick = "document.getElementById('content6').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p><!--/html_preserve--></div></div>
    </form>
  </div>
  
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-8"><div class="rcode"><!--html_preserve--><h4>Example Exploratory Analysis: Amalgamations</h4>
<ul>
  <li>Analysis of 64 pancreas FFPE samples</li>
  <ul>
    <li>2 diseased (cancer) and 2 normal pancreas samples, each replicated 16 times</li>
  </ul>
  <li>HTG EdgeSeq Immuno-Oncology Assay</li>
  <ul>
    <li>549 genes</li>
    <li>10 functional groups</li>
  </ul>
</ul>
<div class="col-sm-6">
  <img src="./ASA_Poster_Content/images/Celgene_IO_NPandDP_PCA_Amalg_CLR_NPDP_Identified.png" style="width:100%">
</div>
<div class="col-sm-6">
  <img src="./ASA_Poster_Content/images/Celgene_IO_PCA_CLR_NPDP_AdhesionProbes_.png" style="width:100%">
</div>
<p style="font-size:10px">
  Figure 6. Exploratory analysis of diseased and non-diseased pancreas samples. Left: PCA of the 10 amalgamated functional groups showing separation along the adhesion group. Right: PCA of the probes which constitute the adhesion group showing strong association with MUC1 and FN1.                     <a href = "javascript:void(0)" onclick = "document.getElementById('content7').style.display='block';document.getElementById('fade').style.display='block'">Expand</a>
</p>

<!--/html_preserve--></div></div>
    
    </form>
  </div>
  <div class="col-sm-3">
    <form class="well" style="height:410px;">
<div class="chunk" id="unnamed-chunk-9"><div class="rcode"><!--html_preserve--><h4>Discussion and Future Directions</h4>
<h5>Discussion</h5>
<ul style="font-size:12px">
  <li>Accounting for the compositional nature of next-generation sequencing data:</li>
    <ul>
      <li>Guards against spurious results</li>
      <li>Opens new avenues for statistical process control and batch effect detection</li>
      <li>Allows for dimension reduction and exploratory analyses</li>
    </ul>
</ul>
<hr>
<h5>Future Directions</h5>
<ul style="font-size:12px">
  <li>Leverage compositional data methods for developing classifiers</li>
  <li>Investigate the use of housekeeping or control probes to improve analyses</li>
  <li>Assess the impact of log-ratio transformation choice</li>
</ul>
<hr>
<a href = "javascript:void(0)" onclick = "document.getElementById('content8').style.display='block';document.getElementById('fade').style.display='block'">Literature Cited</a><!--/html_preserve--></div></div>
<!--    <p> <a href = "javascript:void(0)" onclick = "document.getElementById('content8').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>
-->
    </form>
  </div>
</div>

<!--Expanded Contents-->
<div id="content1" class="white_content">
<div class="chunk" id="unnamed-chunk-10"><div class="rcode"><!--html_preserve-->
<h4 align="center">Next-Generation Sequencing Data is Compositional</h4>
<div class="col-sm-4">
  <form class="well" style="height:600px;">
    <h4>Background</h4>
    <p>It is broadly accepted that next-generation sequencing measurements provide only <i>relative</i> abundances of transcripts (Robinson and Smyth 2007; Anders and Huber 2010; Robinson and Oshlack 2010; Law et al. 2014; Lovell et al. 2015). The relative nature of next-generation sequencing has important consequences for downstream analyses.</p>
    <ul>
      <li>NGS technologies produce <i>relative abundances</i> of transcripts</li>
        <ul>
          <li>Finite sample volume from which DNA/RNA is extracted</li>
          <li>Finite number of reads available on each platform</li>
          <li>Read depth varies from run-to-run and sample-to-sample</li>
        </ul>
      <li>Relative abundance implies data arises as a composition</li>
        <ul>
            <li>More reads allocated to a transcript results if fewer reads allocated to other transcripts</li>
            <li>Induces a dependency in the data (Fig. 1)</li>
        </ul>
    </ul>
    
  </form>
</div>
<div class="col-sm-4">
  <form class="well" style="height:600px;">
    <h4>NGS Workflow</h4>
    <img src="ASA_Poster_Content/images/NGS_Workflow.png" style="width:400px;height:400px;">
    <p>
      Figure a. Schematic of the next-generation sequencing workflow.  The number of clusters able to form on the flow cell limits the number of reads available on a sequencing run (Kircher and Kelso 2010). Figure source: <a href="http://www.illumina.com/content/dam/illumina-marketing/documents/products/illumina_sequencing_introduction.pdf">http://www.illumina.com/content/dam/illumina-marketing/documents/products/illumina_sequencing_introduction.pdf</a>
    </p>
  </form>
</div>
<div class="col-sm-4">
  <form class="well" style="height:600px;">
  </form>
</div>
<!--/html_preserve--></div></div>
    <a href = "javascript:void(0)" onclick = "document.getElementById('content1').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content2" class="white_content">
<div class="chunk" id="unnamed-chunk-11"><div class="rcode"><!--html_preserve--><h4 align="center">Challenges of Compositional Data</h4>
<div class="col-sm-4">
  <form class="well" style="height:600px;">
  <p style="font-size:10px">Relative abundance data is subject to an arbitrary sum constraint, \(\sum_{i=1}^{D} t_i = \mathbb{T} \), where \(t_i\) gives the number of reads to sample \(i\) in a sequencing run with \(D\) samples.</p>
    <ol type="I" style="font-size:10px">
      <li>The sum constraint, \(\mathbb{T}\), changes between platforms, sequencing runs, and samples.</li>
      <li>This leads to an arbitrary difference in <i>scale</i> between samples which various normalization methods have sought to correct.</li>
      <li>This also leads to a dependency between the realized counts in NGS data.
    </ol>
    <div class="row">
      <div class="col-sm-6">    
      <img src="./ASA_Poster_Content/images/BarAnimation.gif" alt="HTML5 Icon" style="width:200px;height:120px;"> 
      </div>
      <div class="col-sm-6">
       <p style="font-size:10px">
       <b>Figure 1:</b> Changes to the absolute abundance of a single transcript changes the relative abundance of the other transcripts being measured due to a total sum constraint. Ignoring the constraints imposed by compositional data can lead to spurious results.
       </p>
      </div>
    </div>
    <div class="row">
      <div class="col-sm-6">    
      <img src="./ASA_Poster_Content/images/DiffAnimation.gif" alt="HTML5 Icon" style="width:200px;height:120px;"> 
      </div>
      <div class="col-sm-6">
       <p style="font-size:10px">
       <b>Figure 2:</b> This animation shows the spurious results one might obtain from relative abundance data if it were treated as absolute abundance data.  As opposed to identifying the 1 true up-regulated mRNA, the observer would also incorrectly identify the other 3 mRNA as down-regulated.
       </p>
      </div>
    </div>
  <p style="font-size:10px">Compositions lie in the \(D-1\) dimensional simplex (\(S^{D-1}\))</p>
    <!--<ol type="I" style="font-size:10px">
      <li>Complicated covariance structure</li>
    </ol>-->
    <img src="./ASA_Poster_Content/images/SimplexGeometry.png" style="width:300px;height:125px>">
  </form>
</div><!--/html_preserve--></div></div>
    <a href = "javascript:void(0)" onclick = "document.getElementById('content2').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content3" class="white_content">
    This is the expanded Content 3. 
    <a href = "javascript:void(0)" onclick = "document.getElementById('content3').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content4" class="white_content">
    This is the expanded Content 4. 
    <a href = "javascript:void(0)" onclick = "document.getElementById('content4').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>

<div id="content5" class="white_content">
<div class="chunk" id="unnamed-chunk-12"><div class="rcode"><!--html_preserve-->

<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h4 align="center">Composition-Based Quality Control</h4>
    <p>We capitalize on the compositional nature of RNA-seq data to develop a quality control metric for sequencing runs</p>
    <ul>
      <li>The number of reads/sample is constrained by total reads/sequencing run.</li>
      <li>We expect the reads to be proportionally allocated across samples.  Samples with unusually low or high reads indicate a possible problem with the sample preparation or sequencing run</li>
      <li>CLR transformation allows the use of standard outlier detection methods (Cook's Distance/ GOF).</li>
    </ul>
    <img src="./ASA_Poster_Content/images/CLR_QC.png" style="width:100%;">
    
      <p><b>Figure 4.</b>Heatmap of CLR transformed reads showing the samples with a very small proportion of the total reads in a sequencing run.</p>
  </form>  
</div>

<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h4 align="center">Batch Effect Detection (without normalization)</h4>
    <p>
    Batch effects are technical variation that can possibly confound biologic variation
    <ul>
      <li>Typical methods for detection of batch effects:
        <ul>
          <li>Multivariate methods - Principle Components Analysis or Hierarchical Clustering</li>
          <li>Visual inspection</li>
        </ul>
          
      <li>Traditional batch effect detection with multivariate methods requires prior sample normalization</li>
        <ul>
          <li>Requires multiple samples and must be re-normalized when a new sample is added</li>
          <li>Generally assumes most transcripts are equally expressed</li>
          <li>Euclidean distance can be misleading when measuring distances between compositions (Aitchison 1985)</li>
        </ul>
      <li>CLR transformation permits batch effect detection using PCA without additional normalization</li>
        <ul>
          <li>CLR covariances are interpretable in \( R^k \)</li>
          <li><b>Aitchison distance</b> (1986) provides compositional equivalent to Euclidean distance</li>
          \[ d_A(x, y) = ||clr(x) - clr(y)||_2 \]
          <li>Enables the use of methods which utilize correlations or distances (e.g. PCA, clustering, outlier detection)</li>
          <li>Applied to individual samples - no need to re-transform if new samples are added</li>
          <li>No assumption about the similarity of samples (e.g. most transcripts are equally expressed)</li>
        </ul>
    </ul>    
    
    </p>
    
  </form>  
</div>
<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h5 align="center">PCA on log(Raw) Read Counts</h5>
    <img src="./ASA_Poster_Content/images/miRNA_Repro_PCA_log(raw).png" style="width:75%;display:block;margin:auto">
    <h5 align="center">PCA on CLR Transformed Read Counts</h5>
    <img src="./ASA_Poster_Content/images/miRNA_Repro_PCA_CLR.png" style="width:75%;display:block;margin:auto">
    <p style="font-size:10px">Principle components analysis of log-counts (top) and centered log-ratio (CLR) transformed counts (bottom).  Although no batch effects are detected in these data, the CLR transformation removes unwanted variation allowing clear separation of the sample types.</p>
  </form>
  <p align="right">
    <a href = "javascript:void(0)" onclick = "document.getElementById('content5').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
  </p>
</div>
  
    <!--/html_preserve--></div></div>
    
</div>

<div id="content6" class="white_content">
<div class="chunk" id="unnamed-chunk-13"><div class="rcode"><!--html_preserve-->

<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h4 align="center">Sub-Compositions</h4>
      <ul>
        <li>Ability to isolate sub-groups of probes from the sequenced gene set for focused analysis</li>
        <li>Can reduce noise from excluded genes with high variation</li>
      </ul>
    <p>The sub-compositional consistency of compositional methods can allow the evaluation of control or housekeeping probes for the evaluation of sample quality</p>
    <img src="./ASA_Poster_Content/images/miRNA_Repro_Proportionality_of_PosCntrl_Probes_byType.png" style="width:100%;display:block;margin:auto">
      
  </form>  
</div>

<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h4 align="center">Amalgamations</h4>
    <ul>
      <li>Can collapse groups of genes with the same functionality into a single proportion</li>
      <li>Facilitates the use of exploratory techniques for finding differences in pathways/functional groups by reducing the dimension of the data</li>
      <li>Amalgamations retain compositional structure</li>
      <li>Examples of hierarchical amalgamations:</li>
        <ul>
          <li>Total reads over entire run</li>
          <li>Sample level total reads</li>
          <li>Reads assigned to functional groups of transcripts (Housekeeping, oncogenes, etc.)</li>
        </ul>
    </ul>
    <hr width="70%">
    
    <p>HTG Molecular Immuno-Oncology (IO) consists of 549 probes which can be amalgamated into 10 functional groups (right).  This dimensional reduction allows the use of visual exploratory methods such as PCA or clustering (see example analysis).</p>
  </form>  
</div>

<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <img src="./ASA_Poster_Content/images/IO gene list overview.png" style="height:100%;display:block;margin:auto;background-color:white">
    
  </form>
  <p align="right">
    <a href = "javascript:void(0)" onclick = "document.getElementById('content6').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
  </p>
</div>
  
    <!--/html_preserve--></div></div>
    
</div>

<div id="content7" class="white_content">
<div class="chunk" id="unnamed-chunk-14"><div class="rcode"><!--html_preserve-->
<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h4 align="center">Example Exploratory Analysis: Amalgamations</h4>
    <ul style="font-size:12px">
      <li>Analysis of 64 pancreas FFPE samples</li>
        <ul>
          <li>2 diseased (cancer) and 2 normal pancreas samples, each replicated 16 times</li>
        </ul>
      <li>HTG EdgeSeq Immuno-Oncology Assay</li>
        <ul>
          <li>549 genes</li>
          <li>10 functional groups</li>
        </ul>
    </ul>
    <h5 align="center">Initial Amalgamation</h5>
    <img src="./ASA_Poster_Content/images/Celgene_IO_NPandDP_PCA_Amalg_CLR_NPDP_Identified_big.png" style="width:100%">
    <p style="font-size:12px">
      Figure 6a: Principle components analysis (covariance bi-plot) of the 10 amalgamated groups.  By reducing the dimension of the 549  probes through a scientifically derived hierarchical amalgamation we are able to determine which functional groups of probes are associated with the separation of healthy and diseased pancreas samples.  From this plot we identify adhesion and immune function groups for further analysis.    
    </p>
  </form>  
</div>

<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h5 align="center">Adhesion Group</h5>
    <img src="./ASA_Poster_Content/images/Celgene_IO_PCA_CLR_NPDP_AdhesionProbes_big.png" style="height:38%;display:block;margin:auto">
    <p style="font-size:12px">
      Figure 6b: Covariance bi-plot of the 79 genes in the cell adhesion functional group showing both MUC1 and FN1 genes associated with separation of diseased and normal samples. 
    </p>
    <img src="./ASA_Poster_Content/images/Celgene_IO_CLR_NPDP_AdhesionProbes_Diff(sidebyside).png" style="height:38%;display:block;margin:auto">
    <p style="font-size:12px">
      Figure 6c: Average of the CLR transformed reads for the cell adhesion functional group genes.  The two genes identified by the covariance bi-plot, MUC1 and FN1, exhibit the greatest difference in proportions between diseased and normal samples. 
    </p>
  </form>  
</div>

<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h5 align="center">Immune Function Group</h5>
      <img src="./ASA_Poster_Content/images/Celgene_IO_PCA_CLR_NPDP_ImmunFuncProbes.png" style="height:38%;display:block;margin:auto">
      <p style="font-size:12px">
        Figure 6d: Covariance bi-plot of the 15 genes in the immune function group showing the DEFB1 gene associated with the separation of diseased and normal samples.  
      </p>
      <img src="./ASA_Poster_Content/images/Celgene_IO_CLR_NPDP_ImmunFuncProbes_Diff(sidebyside).png" style="height:38%;display:block;margin:auto">
      <p style="font-size:12px">
        Figure 6e: Average of the CLR transformed reads for the immune function group of genes.  The largest difference in proportion between diseased and normal samples is in the DEFB1 gene. 
      </p>
  </form>
  <p align="right">
    <a href = "javascript:void(0)" onclick = "document.getElementById('content7').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
  </p>
</div>
  
    <!--/html_preserve--></div></div>
</div>

<div id="content8" class="white_content">
<div class="chunk" id="unnamed-chunk-15"><div class="rcode"><!--html_preserve-->

<form class="well" style="height:700px;">
  <h4>Literature Cited</h4>
  <ul >
    <li style="margin-bottom:20px">Aitchison, J (1985). The statistical analysis of compositional data. Chapman & Hall, Ltd. isbn: 0-412-28060-4. url: <a href="http://dl.acm.org/citation.cfm?id=17272">http://dl.acm.org/citation.cfm?id=17272</a></li>
    
    <li style="margin-bottom:20px">Anders, Simon and W Huber (2010). Differential expression analysis for sequence count data". In: GenomeBiol 11.10, R106. issn: 1465-6906. doi: 10.1186/gb-2010-11-10-r106. url: <a href="http://www.biomedcentral.com/content/pdf/gb-2010-11-10-r106.pdf">http://www.biomedcentral.com/content/pdf/gb-2010-11-10-r106.pdf</a></li>
    
    <li style="margin-bottom:20px">Kircher, Martin and J Kelso (2010). High-throughput DNA sequencing – concepts and limitations. Bioessays 32: 424-536. url: <a href="http://barleyworld.org/sites/default/files/d912f51131ed787b91.pdf">http://barleyworld.org/sites/default/files/d912f51131ed787b91.pdf</a></li>
    
    <li style="margin-bottom:20px">Law, Charity W et al. (2014). voom: Precision weights unlock linear model analysis tools for RNA-seq read counts." En. In: Genome biology 15.2, R29. issn: 1465-6914. doi: 10.1186/gb-2014-15-2-r29. url: <a href="http://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-2-r29">http://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-2-r29</a></li>
    
    <li style="margin-bottom:20px">Lovell, David et al. (2011). \Proportions, Percentages, PPM: Do The Molecular Biosciences Treat Compositional Data Right?" In: Compositional Data Analysis: Theory and Applications. October. John Wiley & Sons, Ltd, pp. 191{207. isbn: 9780470711354. doi: 10 . 1002 / 9781119976462 . ch14. url: <a href="http://dx.doi.org/10.1002/9781119976462.ch14">http://dx.doi.org/10.1002/9781119976462.ch14</a></li>
    
    <li style="margin-bottom:20px">Lovell, David et al. (2015). Proportionality: A Valid Alternative to Correlation for Relative Data." In: PLoS computational biology 11.3, e1004075. issn: 1553-7358. doi: 10.1371/journal.pcbi.1004075. url: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25775355">http://www.ncbi.nlm.nih.gov/pubmed/25775355</a></li>
    
    <li style="margin-bottom:20px">Pearson, Karl (1896). Mathematical Contributions to the Theory of Evolution.{On a Form of Spurious Correlation Which May Arise When Indices Are Used in the Measurement of Organs. In: Proceedings of the Royal Society of London 60, pp. 489-498. url: <a href="https://archive.org/details/philtrans00847732">https://archive.org/details/philtrans00847732</a></li>

  </ul>
  
  <a href = "javascript:void(0)" onclick = "document.getElementById('content8').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</form>  <!--/html_preserve--></div></div>
    <a href = "javascript:void(0)" onclick = "document.getElementById('content8').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
</div>
<div id="fade" class="black_overlay"></div>


<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
