

<h4>Next-Generation Sequencing Data is Compositional</h4>
    <p style="font-size:10px">Next-generation sequencing measurements provide only <i>relative</i> abundances of transcripts (Robinson and Smyth 2007; Anders and Huber 2010; Robinson and Oshlack 2010; Law et al. 2014; Lovell et al. 2015). The relative nature of next-generation sequencing has important consequences for downstream analyses.</p>
    <ul style="font-size:10px">
      <li>There are several reasons NGS data are relative.</li>
          <ul>
            <li>Finite sample volume from which DNA/RNA is extracted</li>
            <li>Finite number of reads available on each platform (as the number of reads increases for 1 transcript the number of available reads for the other transcripts decreases, Fig. 1)</li>
            <li>Read depth (number of reads allocated to a sample) varies from run-to-run and sample-to-sample</li>
          </ul>
<!-- This might be more appropriate somewhere else          
      <li>The number of reads for a given transcript can be affected by a number of <i>experiment level</i> factors including (Sims et al. 2014):</li>
        <ul>
          <li>Read depth of the sequencing run (sequencing platform)</li>
          <li>Number of samples included in a sequencing run</li>
          <li>Volume of sample input</li>
        </ul>
-->
      <li>Traditional methods can produce misleading results for compositional data</li>
        <ul>
          <li>Spurious correlation (Pearson...)</li>
          <li>Univariate results may be misleading (Lovell et al. 2015)</li>
          <li>Negative bias of the covariance matrix</li>
        </ul>
      
    </ul>
<p align="right"> <a href = "javascript:void(0)" onclick = "document.getElementById('content1').style.display='block';document.getElementById('fade').style.display='block'">Expand</a></p>




