

<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h4 align="center">Composition-Based Quality Control</h4>
    <p>We capitalize on the compositional nature of RNA-seq data to develop a quality control metric for sequencing runs</p>
    <ul>
      <li>The number of reads/sample is constrained by total reads/sequencing run.</li>
      <li>We expect the reads to be proportionally allocated accross samples.  Samples with unusully low or high reads indicate a possible problem with the sample preparation or sequencing run</li>
      <li>CLR transformation allows the use of standard outlier detection methods (Cook's Distance/ GOF).</li>
    </ul>
    <img src="./ASA_Poster_Content/images/CLR_QC.png" style="width:100%;">
    
      <p><b>Figure 4.</b>Heatmap of CLR transformed reads showing the samples with a very small proportion of the total reads in a sequencing run.</p>
  </form>  
</div>

<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h4 align="center">Batch Effect Detection (without normalization)</h4>
    <p>
    Batch effects are technical variation that can possibly confound biologic variation
    <ul>
      <li>Typical methods for detection of batch effects:
        <ul>
          <li>Multivariate methods - Principle Components Analysis or Hierarchical Clustering</li>
          <li>Visual inspection</li>
        </ul>
          
      <li>Traditional batch effect detection with multivariate methods requires prior sample normalization</li>
        <ul>
          <li>Requires multiple samples and must be re-normalized when a new sample is added</li>
          <li>Generally assumes most transcripts are equally expressed</li>
          <li>Euclidean distance can be misleading when measuring distances between compositions (Aitchison 1985)</li>
        </ul>
      <li>CLR transformation permits batch effect detection using PCA without additional normalization</li>
        <ul>
          <li>CLR covariances are interpretable in \( R^k \)</li>
          <li><b>Aitchison distance</b> (1986) provides compositional equivalent to Euclidean distance</li>
          \[ d_A(x, y) = ||clr(x) - clr(y)||_2 \]
          <li>Enables the use of methods which utilize correlations or distances (e.g. PCA, clustering, outlier detection)</li>
          <li>Applied to individual samples - no need to re-transform if new samples are added</li>
          <li>No assumption about the similarity of samples (e.g. most transcripts are equally expressed)</li>
        </ul>
    </ul>    
    
    </p>
    
  </form>  
</div>
<div class="col-sm-4">
  <form class="well" style="height:700px;">
    <h5 align="center">PCA on log(Raw) Read Counts</h5>
    <img src="./ASA_Poster_Content/images/miRNA_Repro_PCA_log(raw).png" style="width:75%;display:block;margin:auto">
    <h5 align="center">PCA on CLR Transformed Read Counts</h5>
    <img src="./ASA_Poster_Content/images/miRNA_Repro_PCA_CLR.png" style="width:75%;display:block;margin:auto">
    <p style="font-size:10px">Principle components analysis of log-counts (top) and centered log-ratio (CLR) transformed counts (bottom).  Although no batch effects are detected in these data, the CLR transformation removes unwanted variation allowing clear separation of the sample types.</p>
  </form>
  <p align="right">
    <a href = "javascript:void(0)" onclick = "document.getElementById('content5').style.display='none';document.getElementById('fade').style.display='none'">Close</a>
  </p>
</div>
  
    
