
<h4 align="center">Next-Generation Sequencing Data is Compositional</h4>
<div class="col-sm-4">
  <form class="well" style="height:600px;">
    <h4>Background</h4>
    <p>It is broadly accepted that next-generation sequencing measurements provide only <i>relative</i> abundances of transcripts (Robinson and Smyth 2007; Anders and Huber 2010; Robinson and Oshlack 2010; Law et al. 2014; Lovell et al. 2015). The relative nature of next-generation sequencing has important consequences for downstream analyses.</p>
    <ul>
      <li>NGS technologies produce <i>relative abundances</i> of transcripts</li>
        <ul>
          <li>Finite sample volume from which DNA/RNA is extracted</li>
          <li>Finite number of reads available on each platform</li>
          <li>Read depth varies from run-to-run and sample-to-sample</li>
        </ul>
      <li>Relative abundance implies data arises as a composition</li>
        <ul>
            <li>More reads allocated to a transcript results if fewer reads allocated to other transcripts</li>
            <li>Induces a dependency in the data (Fig. 1)</li>
        </ul>
    </ul>
    <img src="./ASA_Poster_Content/images/BarAnimation.gif" alt="HTML5 Icon" style="display:block;margin:auto;width:220px;height:140px;">
  </form>
</div>
<div class="col-sm-4">
  <form class="well" style="height:600px;">
    <h4>NGS Workflow</h4>
    <img src="ASA_Poster_Content/images/NGS_Workflow.png" style="width:400px;height:400px;">
    <p>
      Figure a. Schematic of the next-generation sequencing workflow.  The number of clusters able to form on the flow cell limits the number of reads available on a sequencing run (Kircher and Kelso 2010). Figure source: <a href="http://www.illumina.com/content/dam/illumina-marketing/documents/products/illumina_sequencing_introduction.pdf">http://www.illumina.com/content/dam/illumina-marketing/documents/products/illumina_sequencing_introduction.pdf</a>
    </p>
  </form>
</div>
<div class="col-sm-4">
  <form class="well" style="height:600px;">
    <h4>Another Common Relative Frequency Measure: RT-qPCR</h4>
    <ui>
      <li>Analysis of qPCR is usually conducted through \(\delta \delta\) CT values</li>
        <ul>
          <li>\(\delta \delta\)CT is a <i>relative quantitation method</i></li>
          <li>CT is a quantitation of transcipt amount</li>
          <li>A Reference Gene is used to normalize possible variations in CT values from:</li>
            <ul>
              <li>Sample prep</li>
              <li>RNA isolation</li>
              <li>Reverse transcription efficiency, etc.</li>
            </ul>
        </ul>
    </ui>
    <img src="./ASA_Poster_Content/images/RTqPCR_example.png" style="width:100%;display:block;margin:auto">
    <p style="font-size:13">From <a href="http://sabiosciences.com/manuals/IntrotoqPCR.pdf">http://sabiosciences.com/manuals/IntrotoqPCR.pdf</a></p>
  </form>
</div>

